server/app/services/MessageKey.java

```diff
@@ -22,6 +22,8 @@ public enum MessageKey {
   ADDRESS_VALIDATION_NO_PO_BOX("validation.noPoBox"),
   ADDRESS_VALIDATION_STATE_REQUIRED("validation.stateRequired"),
   ADDRESS_VALIDATION_STREET_REQUIRED("validation.streetRequired"),
+  ALERT_CREATE_ACCOUNT("alert.createAccount"), // North Star only
```

Comment:
**question:** What's the purpose of these "North Star only" comments? I see there's already an established pattern but I'm just curious.

Reply:
When I started, I thought we'll want to delete the old strings after we launch North Star. Annotating the new strings with "North Star only" would make it easier to clean up this file later. A few weeks later, I realized it's probably unnecessary, but I kept doing it out of habit.

I had half an idea to send a PR deleting all the "North Star only" comments. Not sure if it matters, but I'm ok either way.